Source/ACE.Server/Entity/Aetheria.cs

```diff
@@ -133,13 +142,18 @@ public static void UseObjectOnTarget(Player player, WorldObject source, WorldObj
 
             actionChain.AddAction(player, () => ActivateSigil(player, source, target));
 
+            player.EnqueueMotion(actionChain, MotionCommand.Ready);
```

Comment:
player.IsBusy = false should be after the return to ready state animation

IsBusy=true/false should also be kept together in the same function for consistency and clarity